pwiz/analysis/spectrum_processing/SpectrumListFactory.cpp

```diff
@@ -668,6 +688,26 @@ UsageInfo usage_precursorRecalculation = {"","This filter recalculates the precu
     "although it does not use any 3rd party (vendor DLL) code.  Since the time the code was written, Thermo has since fixed "
     "up its own estimation in response, so it's less critical than it used to be (though can still be useful)."};
 
+inline void unescapeQuotedPath(string &path)
+{
+    if ((bal::starts_with(path, "\'") && bal::ends_with(path, "\'")) || (bal::starts_with(path, "\"") && bal::ends_with(path, "\"")))
+    {
+        // Remove the first and last characters
+        path = path.substr(1, path.length() - 2);
+    }
+    // Look for the ^ escape character
+    for (size_t escapePos = 0;;)
```

Comment:
Is this necessary? I would expect the Windows shell to take the `^`s out (but not the escaped quotes). That's the case with that ugly titleMaker filter escaping I sent to you.

Reply:
This is for dealing with what's between the nested quotes, which windows shell doesn't mess with. But I suppose that implies that we maybe didn't need to escape any of that stuff while constructing the command line. I'll have to review.

Reply:
You DO need to escape it when making the command-line in Skyline, but I'm hoping you don't need to process it inside the called exe (msconvert). Here's the titleMaker escaping even though it's inside the nested quotes:
```
--filter "titleMaker <RunId>.<ScanNumber>.<ScanNumber>.<ChargeState> File:"""^<SourcePath^>""", NativeID:"""^<Id^>""""
```
What msconvert sees: 
```
titleMaker <RunId>.<ScanNumber>.<ScanNumber>.<ChargeState> File:"<SourcePath>", NativeID:"<Id>"
```
But I concede it could be different if it's nested inside another level of quoting.

Reply:
The escape characters should not show up inside the program. They are to direct the shell to give the correct characters to the program. If you handle them again side the program, you will be double-escaping, I think.

Reply:
double-unescaping, I meant